### PR TITLE
default py3 dependencies

### PR DESCRIPTION
**Pull request by marcelotduarte**

### CODE REVIEW

**Comment by Berserker66** [comment]
Would this mean no VC Runtime has to be installed seperately and can safely be removed from setup scripts?

**Comment by marcelotduarte** [comment]
Oops ... if this happens, I think I did more than I wanted to, but let's wait for what tuininga will say.

**Comment by marcelotduarte** [comment]
PR #512 fix the python3.dll dependency with pyqy5 sample...

File: cx_Freeze/freezer.py [hunk 1]
```diff
@@ -221,9 +221,10 @@ def _GetDefaultBinIncludes(self):
         """Return the file names of libraries which must be included for the
            frozen executable to work."""
         if sys.platform == "win32":
-            pythonDll = "python%s%s.dll" % sys.version_info[:2]
-            return [pythonDll, "gdiplus.dll", "mfc71.dll", "msvcp71.dll",
-                    "msvcr71.dll"]
+            pythonDlls = ["python%s.dll" % sys.version_info[0],
+                          "python%s%s.dll" % sys.version_info[:2],
+                          "vcruntime140.dll"]
+            return pythonDlls
```
Comment (anthony-tuininga) [inline cx_Freeze/freezer.py:227]: Why drop gdiplus.dll, mfc71.dll, msvcp71.dll and msvcr71.dll? Why not just add vcruntime140.dll? And how does this work with include_mscvcr? I'm not a regular Windows user so can you explain what the intent of this is? Thanks!

Comment (marcelotduarte) [reply cx_Freeze/freezer.py:227]: Py3 do not use this dlls.

**Pull request merged**